github/pulls_test.go

```diff
@@ -232,31 +233,63 @@ func TestPullRequestsService_Create_invalidOwner(t *testing.T) {
 }
 
 func TestPullRequestsService_Edit(t *testing.T) {
-	setup()
-	defer teardown()
+	tests := []struct {
+		input        *PullRequest
+		sendResponse string
+
+		wantUpdate   *pullRequestUpdate
+		wantResponse *PullRequest
+	}{
+		{
+			input:        &PullRequest{Title: String("t")},
+			sendResponse: `{"number":1}`,
+			wantUpdate:   &pullRequestUpdate{Title: String("t")},
```

Comment:
Then, the value for `wantUpdate` here would be something like `{"title":"t"}`.

It's easier to reason about it being correct, since GitHub API deals with JSON input and we can compare its documentation with this.